server/routerlicious/packages/protocol-base/README.md

```diff
@@ -1,7 +1,8 @@
 # @fluidframework/protocol-base
 
-Shared protocol code for client and service to share. Manages the lifetime of Quorum and proposals that needs to be
-consistent across client and service. It also provides utilities for facilitating summary creation.
+## Purpose
+
+The purpose of @fluidframework/protocol-base is to manage the lifetime of Quorum and proposals that needs to be consistent across client and service and git implementation which takes in a summary object and returns it mode/type.
```

Comment:
Nit: granted, I'm not close to this area of FF, but I'm having a hard time making sense of this. The "git implementation" bit is a separate second purpose, right? We're not supposed to read this as "needs to be consistent across client AND service AND git implementation"?

Reply:
Updated the README.